src/modules/markets/entities/market.entity.ts

```diff
@@ -59,11 +59,11 @@ export class MarketEntity {
 
   @Column({ name: 'ESRA_ID' })
   @ApiProperty({ example: 1, description: '' })
-  ESRAClasificationId: number;
+  ESRAClassificationId: number;
 
   @Column({ name: 'ESRA_DESC' })
   @ApiProperty({ example: 'Standard Risk', description: '' })
-  ESRAClasificationDesc: string;
+  ESRAClassificationDesc: string;
```

Comment:
same as above, @avaitonis  can you check if this is correct? there a typo here

Reply:
This typo was introduced 7 months ago and endpoint is now used by EXIP. We can fix typo but release of this nodeJS API will require small change in EXIP project.